tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,737 @@
+                    .setRoot(Paths.get(descriptor.getOutputRealmLocation(), descriptor.getFileSystemRealmName()))
+                    .setSecretKey(key)
+                    .setLevels(descriptor.getLevels())
+                    .setProviders(ELYTRON_PASSWORD_PROVIDERS)
+                    .build();
+            FileSystemRealmUtil.createEncryptedRealmFromUnencrypted(oldFileSystemRealm, newFileSystemRealm);
+        }
+    }
+
+    /**
+     * Creates the script/commands the user must run for Elytron to recognize
+     * and use the new filesystem-realm
+     */
+
+    private void createWildFlyScript() throws Exception {
+        int counter = 0;
+        for (Descriptor descriptor : descriptors) {
+            String inputRealmLocation = descriptor.getInputRealmLocation();
+            String outputRealmLocation = descriptor.getOutputRealmLocation();
+            String fileSystemRealmName = descriptor.getFileSystemRealmName();
+            String credentialStore = descriptor.getCredentialStore();
+            String secretKeyAlias = descriptor.getSecretKeyAlias();
+            int levels = descriptor.getLevels();
+            Charset hashCharset = descriptor.getHashCharset();
+
+            if(secretKeyAlias == null) {
+                secretKeyAlias = "key";
+            }
+            String createScriptCheck = "";
+            if (Paths.get(String.format("%s.sh", fileSystemRealmName)).toFile().exists()) {
+                createScriptCheck = prompt(
+                        false,
+                        null,
+                        false,
+                        ElytronToolMessages.msg.shouldFileBeOverwritten(String.format("%s.sh", fileSystemRealmName))
+                );
+            }
+            String fullOutputPath;
+            if (outputRealmLocation.startsWith(".")) {
+                fullOutputPath = Paths.get(outputRealmLocation.substring(2, outputRealmLocation.length())).toAbsolutePath().toString();
+            } else {
+                fullOutputPath = Paths.get(outputRealmLocation).toAbsolutePath().toString();
+            }
+
+            if (summaryMode) {
+                summaryString.append(String.format("Configured script for WildFly named %s.sh at %s.", fileSystemRealmName, fullOutputPath));
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append("The script is using the following names:");
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append(String.format("Name of filesystem-realm: %s", fileSystemRealmName));
+                summaryString.append(System.getProperty("line.separator"));
+            }
+
+            List<String> scriptLines = Arrays.asList(
+                String.format("/subsystem=elytron/secret-key-credential-store=%s:add(path=%s, create=true)", "mycredstore"+counter, fullOutputPath),
```

Comment:
@Ashpan Just minor, the  path to `secret-key-credential-store` is not the path to the cred store but the path to the realm here.

Reply:
I set the path to be at the realm location because if it's being used in bulk mode, I wanted it to be clear which credential store is used for which filesystem realm. 
Changed the path to include the cred store name too: `fullOutputPath+"/mycredstore.cs"`